kafka-testing/src/test/resources/kafka/produce-consume/test_kafka_protobuf.json

```diff
@@ -0,0 +1,58 @@
+{
+	"scenarioName": "Produce and consume a protobuf message",
+	"steps": [
+		{
+			"name": "produce_protobuf_person",
+			"url": "kafka-topic:demo-protobuf-topic-1",
+			"operation": "produce",
+			"request": {
+				"recordType": "PROTO",
+				"protobufMessageClassType": "org.jsmart.zerocode.proto.PersonsProto$Person",
```

Comment:
Does this have to match with `value.serializer=org.jsmart.zerocode.proto.serialization.PersonSerializer` ?
Just checking...

Reply:
Yes i looked at it but there is how protobuf works and rhis is why it is so optimized i couldnt find an easy way to write a generic protobuf